res/values-pt-rBR/strings.xml

```diff
@@ -1166,7 +1166,7 @@ Esta é uma lista de coisas que é * PROIBIDO * para usar nas skins personalizad
     <string name="Tricks_Performed_">Tricks Realizados:</string>
     <string name="Not_enough_plasma_">Plasma insuficiente!</string>
     <string name="GET_PLASMA">OBTER PLASMA</string>
-    <string name="ASTEROIDS">ASTEROIDES</string>
+    <string name="ASTEROIDS">ASTERÓIDES</string>
```

Comment:
According to current Portuguese grammatical rules, "Asteroides" was already correct.